### ISSUE TITLE
Website Issue | Issues when using a screen reader

### ISSUE THREAD

**Issue by gracjankn** [privacyguides/privacyguides.org#159]
## Description

Since links to download software for a certain platform (F-Droid, Google Play Store, Linux, iOS etc.) only have icons, they aren't accessible to users of screen readers, because they are [missing names](https://web.dev/link-name). All you hear _link, link, link_….

If we don't want to add visible text and only keep the icons, the `<i>` tags for icons should have `aria-hidden="true"` and the `<a>` inside them should have `aria-label` set to the name of a platform.

Top navigation shouldn't be nested in the `<header>`. It should be in in `<nav>` tag adjacent to it.

The _legal icons_ in the footer (left side of the line that specifies which license the site is under) are selectable but not spoken. They also need accessibility labels.

The main page has [incorrect order of heading sizes](https://web.dev/heading-order).

**Comment by dngray** [comment]
I wonder if this is fixed now? https://github.com/privacyguides/privacyguides.org/commit/00f18af4659cd6dd628ae3fdedf95b532b7a5e64
